teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(age, true, name)
```

Comment:
- [ ] Also the super function should have (age, name, parent_permission: true) as per requirement unlike what we have

Reply:
Thanks for your review. I will implement this  😄